build.gradle

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.elasticsearch.gradle.ElasticsearchProperties
+
+buildscript {
+  repositories {
+    mavenCentral()
+    maven {
+      name 'sonatype-snapshots'
+      url 'http://oss.sonatype.org/content/repositories/snapshots/'
+    }
+    jcenter()
+  }
+  dependencies {
+    classpath 'org.elasticsearch.gradle:build-tools:3.0.0-SNAPSHOT'
+  }
+}
+apply plugin: 'idea'
+apply plugin: 'eclipse'
+apply plugin: 'elasticsearch.esplugin'
+
+esplugin {
+  description 'The mapper attachments plugin adds the attachment type to Elasticsearch using Apache Tika.'
+  classname 'org.elasticsearch.plugin.mapper.attachments.MapperAttachmentsPlugin'
+}
+
+project.group = 'org.elasticsearch'
+project.version = ElasticsearchProperties.version
```

Comment:
Hrm, I thought we released 2.0 plugins under a different groupId so that we could get the plugin versions in sync with elasticsearch version?